lemur/notifications/messaging.py

```diff
@@ -225,6 +243,14 @@ def send_pending_failure_notification(
     :param notification_plugin:
     :return:
     """
+    function = f"{__name__}.{sys._getframe().f_code.co_name}"
+    log_data = {
+        "function": function,
+        "message": "Sending pending failure notification for pending certificate {}".format(pending_cert.name),
+        "notification_type": "failed",
```

Comment:
How about "pending failed" or "pending certificate failed" as notification_type, something more specific?

Reply:
I used `failed` as it matches the name of the notification type sent to the plugin. I agree it's not the most descriptive name, but I'd also suggest it would be better to keep the name of the notification type consistent everywhere. Maybe we could look into renaming `failed` to something more specific later? I anticipate more notification refactor work coming soon(-ish).